api.go

```diff
@@ -492,9 +492,26 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		}
 		r.processConfigurationLogEntry(&entry)
 	}
+
 	r.logger.Printf("[INFO] raft: Initial configuration (index=%d): %+v",
 		r.configurations.latestIndex, r.configurations.latest.Servers)
 
+	mismatchedIP := false
```

Comment:
This was necessary for raft peers call to show the correct IP address for whoever becomes the leader. Otherwise, even though the network communication between all the peers work, it looks incorrect to the operator.  
I didn't notice configuration being incorrect when i tried doing `consul operator raft list-peers` using the http-addr of each server with this in place. Is there some other way to view configuration that's like a local only view that I am missing?